nautobot_chatops/models.py

```diff
@@ -113,10 +113,6 @@ def __str__(self):
         """String representation of an AccessGrant."""
         return f'cmd: "{self.command} {self.subcommand}", {self.grant_type}: "{self.name}" ({self.value})'
 
-    def get_absolute_url(self, api=False):
```

Comment:
Not for future reviewer. We now have a detail view that we can return to so this is no longer needed.